docsite/index.js

```diff
@@ -1,10 +1,10 @@
-import 'https://cdnjs.cloudflare.com/ajax/libs/prism/1.16.0/prism.min.js'
 import 'https://cdnjs.cloudflare.com/ajax/libs/prism/1.16.0/plugins/normalize-whitespace/prism-normalize-whitespace.min.js'
+import 'https://cdnjs.cloudflare.com/ajax/libs/prism/1.16.0/prism.min.js'
 
-import '/js/easing.js'
 import '/js/animations.js'
-import '/js/slider.js'
+import '/js/brand-colors.js'
+import '/js/color-copy.js'
+import '/js/easing.js'
 import '/js/scrollspy.js'
 import '/js/select.js'
-
-import '/js/color-copy.js'
\ No newline at end of file
+import '/js/slider.js'
```

Comment:
looks like a linter ran here? if nothing changed, can you remove it from the PR? 🙏🏻

Reply:
is it fine now?